src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -77,6 +79,18 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 	});
 };
 
+export const generateBranchName = (issueKey: string, issueSummary: string) => {
+	if (!issueSummary) {
+		return `${issueKey}-`;
```

Comment:
Should we add `-` at the end of the branch name?

Reply:
poooof going